apps/concierge_site/lib/templates/account/edit.html.eex

```diff
@@ -35,6 +35,19 @@
           <%= label form, :phone_number, "My phone number is:", class: "form__label d-block" %>
           <%= telephone_input form, :phone_number, autocomplete: "off", placeholder: "###-###-####", class: "form-control d-inline-block form__phone--input", data: [toggle: "input"], required: false %>
           <%= error_tag form, :phone_number %>
+          <div class="mt-3 form-check">
```

Comment:
suggestion(non-blocking): It looks like this block is repeated in 2 places, so it could get pulled out into a partial if you wanted.

Reply:
yep, was thinking about that but wanted to keep it separate for now in case there were any design differences we wanted.